fabric-tool-attribute-api-v1/src/main/java/net/fabricmc/fabric/api/tool/attribute/v1/DynamicAttributeTool.java

```diff
@@ -33,22 +36,73 @@
 	/**
 	 * Determines the mining level of the passed stack, which is used for calculating what blocks this tool is allowed to break.
 	 *
-	 * @param stack The stack to check on.
-	 * @param user The current user of the tool, or null if there isn't any.
+	 * @param stack The stack mining the block
```

Comment:
```suggestion
	 * @param stack The item stack being used to mine the block
```

Reply:
`The item stack used to mine the block` instead?